Core/src/OffConverter/OffModel.cpp

```diff
@@ -24,13 +24,19 @@ void OpenInfraPlatform::Core::OffConverter::OffModel::reset()
 	geometry_.reset();
 }
 
-void OpenInfraPlatform::Core::OffConverter::OffModel::addVertices(std::vector<buw::Vector3d> vertices)
+void OpenInfraPlatform::Core::OffConverter::OffModel::addVertices(const std::vector<buw::Vector3d>& vertices)
 {
 	for (const auto& vertex : vertices)
 		geometry_.vertices.push_back(vertex);
```

Comment:
It is advisable to use `.reserve(..)` when the amount of data "pushed back" to a `std::vector` is known - it saves a lot of allocations. Beware of the original size to preserve it!

Reply:
Resolved: 6fad8a0